### ISSUE TITLE
add test case

### ISSUE THREAD

**Issue by qct** [qct/bitmex-go#1]
main test is too naive, add formal test case instead.

**Comment by DaKoTaGeorgia** [comment]
Hello, i run with main.go tests Good Job!
but please let mek now how i can start live bitmex trading or it's need build my self?
please let me know

**Comment by qct** [comment]
@DaKoTaGeorgia 
No, you don't need to build yourself.

1. `go get github.com/qct/bitmex-go` 
2. import this repo  like `main.go`: 
```
import (
    "github.com/qct/bitmex-go/restful"
    "github.com/qct/bitmex-go/swagger"
    "golang.org/x/net/context"
    "io/ioutil"
    "log"
)
```
3. then you can do live bitmex trading like what did in `main.go`:
```
orderApi := restful.NewOrderApi(apiClient.OrderApi, auth)
resp, orderId, err := orderApi.LimitBuy("XBTUSD", 1.0, 13000, "qct_f_f_")
```

**Comment by DaKoTaGeorgia** [comment]
yes sorry for noob questions but i now working with simple-bitmex-bot and now i run go on my macbook i run this code good but only test results... i mean when i will run your bot it will trading automatical with included logic or i need write logic myself ?

**Comment by DaKoTaGeorgia** [comment]
in you code: 

orderApi := restful.NewOrderApi(apiClient.OrderApi, auth)
resp, orderId, err := orderApi.LimitBuy("XBTUSD", 1.0, 13000, "qct_f_f_")

bot placing 1 contract 13000$ (i mean how trade automatical...)

**Comment by qct** [comment]
@DaKoTaGeorgia this repo is a SDK, not a strategy, you need to write your own logic code.
